mlir/test/Dialect/Tosa/constant-op-fold.mlir

```diff
@@ -3,6 +3,20 @@
 
 // RUN: mlir-opt --split-input-file --tosa-layerwise-constant-fold="aggressive-reduce-constant=true" %s | FileCheck %s --check-prefix=AGGRESIVE
 
+// CHECK-LABEL: @argmax_fold_dim_1
```

Comment:
nit: I thought from the name this was looking at dimension 1, not a dimension of size 1. maybe @armax_fold_dim_size_1 instead?